src/WebJobs.Script/WorkerUtilities.cs

```diff
@@ -17,5 +17,41 @@ public static int GetUnusedTcpPort()
                 return port;
             }
         }
+
+        /// <summary>
+        /// Determines whether the specified port is available.
+        /// </summary>
+        internal static bool CanBindToPort(int port)
+        {
+            // Try to bind to the port using IPv6 dual mode socket to cover both IPv4 and IPv6.
+            using (var tcpSocket = new Socket(AddressFamily.InterNetworkV6, SocketType.Stream, ProtocolType.Tcp) { DualMode = true })
```

Comment:
Use the new using style:

```suggestion
            using Socket tcpSocket = new(AddressFamily.InterNetworkV6, SocketType.Stream, ProtocolType.Tcp) { DualMode = true };
```